platform/android/MapboxGLAndroidSDK/src/main/AndroidManifest.xml

```diff
@@ -1,8 +1,10 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     package="com.mapbox.mapboxsdk">
 
     <uses-feature android:glEsVersion="0x00020000" android:required="true" />
     <uses-feature android:name="android.hardware.wifi" android:required="false" /> <!-- Implied by ACCESS_WIFI_STATE. -->
+    <uses-feature android:name="android.hardware.location.gps" android:required="false" tools:replace="required" />
```

Comment:
@osana @tobrun Do we need to add `tools:replace="required"` for the merge to work as expected? It didn't seem to be needed for the `android.hardware.wifi` rule.

Reply:
@zugaldia @tobrun I will follow your lead on this.

Reply:
@osana could you run a quick test using the test app? You should be able to see the merged manifest of the app in Studio and verify whether `tools:replace="required"` is needed. Thanks!